omero/sysadmins/config.rst

```diff
@@ -1639,6 +1639,22 @@ The Glacier2 TCP port number to use (unencrypted)
 
 Default: `4063`
 
+.. property:: omero.ports.ws
+
+omero.ports.ws
+^^^^^^^^^^^^^^
+The Glacier2 WS port number to use (unecrypted)
```

Comment:
Typo that should likely be fixed upstream

Reply:
it needs to be fixed upstream

Reply:
This will require https://github.com/ome/omero-py/pull/312 to be merged and a new version of omero-py

Reply:
Is this comment related to https://github.com/ome/omero-documentation/pull/2203#discussion_r767793018 instead?

Reply:
yes. Sorry
This has been fixed